assets/eip-5725/contracts/BaseVestingNFT.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
For file in the `assets` directory, we ask that reference implementations use licenses that don't impose restrictions on downstream projects. `CC0-1.0` is preferred, but I'll also accept MIT / Apache.